### PR TITLE
Create plugin-shell.rst

### PR DESCRIPTION
**Pull request by bobmulder**
Added documentation for the plugin shell.

https://github.com/cakephp/cakephp/pull/6042

### CODE REVIEW

File: en/console-and-shells/plugin-shell.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    
+    
+Loading plugins
+---------------
+
+Via the `Load` task you are able to load plugins in your `config/bootstrap.php`. You can do this by running::
+
+    bin/cake plugin load MyPlugin
+
+This will add the following to your `config/bootstrap.php`::
```
Comment (markstory) [inline en/console-and-shells/plugin-shell.rst]: Backticks need to be doubled for sphinx/RST.

File: en/console-and-shells/plugin-shell.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    bin/cake plugin load -r MyPlugin
+    
+    // will return
+    Plugin::load('MyPlugin', ['bootstrap' => false, 'routes' => true, 'autoload' => true]);
+
+Unloading plugins
+-----------------
+
+You can unload a plugin by specifying it's name::
```
Comment (ravage84) [inline en/console-and-shells/plugin-shell.rst]: "its name"

Comment (bobmulder) [inline en/console-and-shells/plugin-shell.rst]: Thank you!

**Comment by bobmulder** [comment]
Can this be merged becaus of https://github.com/cakephp/cakephp/commit/2107d61c6c244175105a87d3860483096b8e4d20 ?

**Comment by bobmulder** [comment]
The plugin shell page is not showing up yet @markstory.

**Comment by antograssiot** [comment]
@bobmulder it's becuse it's not inserted in any toc.
I've open #2536 this morning to correct this

**Comment by bobmulder** [comment]
Thank You! I did not know about the toc. Sorry

**Pull request merged**